services/upload.js

```diff
@@ -0,0 +1,48 @@
+const azureStorage = require('azure-storage')
+const {
+  azureAccess: { STORAGE_ACCOUNT, ACCESS_KEY, AZURE_HOST }
+} = require('~/configs/config')
+
+let blobService
+
+const uploadService = {
+  uploadFile: async (file, containerName) => {
```

Comment:
i think that key word "async" is redundant here

Reply:
Yep, You are right. I deleted it.